### ISSUE TITLE
bundle install fails to install openstudio-standards 0.2.10

### ISSUE THREAD

**Issue by chriswmackey** [urbanopt/urbanopt-example-geojson-project#18]
It seems that there may be a bug in the installation of openstudio-standards 0.2.10 that happens when I run `bundle install`  per [the instructions here](https://docs.urbanopt.net/usage/run_example.html).
![image](https://user-images.githubusercontent.com/5567574/69459156-226eaa00-0d3f-11ea-93f4-ce20b8e5f330.png)

**Comment by chriswmackey** [comment]
Let me know if this issue belongs on the openstuido-standards repo instead of here.

**Comment by vtnate** [comment]
Thanks Chris, we're trying to replicate the issue here.

**Comment by chriswmackey** [comment]
The log from trying to execute `run_all` might provide some more insight:
![image](https://user-images.githubusercontent.com/5567574/69466663-f9f0ab00-0d52-11ea-9872-5b9be77bae08.png)

**Comment by chriswmackey** [comment]
It might also be helpful to know that, when I run `gem install openstudio-standards`, the process seems to hang indefinitely:
![image](https://user-images.githubusercontent.com/5567574/69466732-489e4500-0d53-11ea-9209-eca963146092.png)

**Comment by tanushree04** [comment]
@chriswmackey This might be due to a long file path for the location where the urbanopt example project is cloned. 
Try using this to allow long path names in git:

 git config --global core.longpaths true

**Comment by mdahlhausen** [comment]
@tanushree04 @chriswmackey we shortened the filenames after the openstudio-standards v0.2.10 release.  I think there may have been some missing .json data as well in constructions and refrigeration.  I'm releasing an openstudio-standards v0.2.11 Thursday which I recommend you use instead.

**Comment by DavidGoldwasser** [comment]
@mdahlhausen do you see any problems using newer gem but with OpenStudio 2.9.1? Do you expect that to break anything measures that worked with 0.2.10 standards gem, or are they minor or just corrective changes?

**Comment by mdahlhausen** [comment]
update: so it looks like the name shortening happened before the v0.2.10 release.  It may just be that the urban opt implementation makes the paths too long.

**Comment by mdahlhausen** [comment]
@DavidGoldwasser I'm updating openstudio-standards to work with OS 3.0.  It already works with OS 2.9.1, though some prototypes are slightly different due to a chiller sizing change between E+ 8.9 and 9.1.

**Comment by chriswmackey** [comment]
I had run `git config --global core.longpaths true` as noted in the [URBANopt installation instrucitons](https://docs.urbanopt.net/installation/windows.html) so that doesn't seem to be the issue here.

I also recently found that I can avoid this error if I copy the geoJSON and mappers of this repo to another folder and run the URBANopt CLI from that folder. The specific reason why I was getting this error when I ran things out of the folder cloned directly from this repo is that URBANopt was automatically running `bundle update` with this repo's gemfile before it went to execute the URBANopt processes. So my guess is that there's something out of sync with what happens when `bundle update` is run from this repo.
